electron/src/main/main.ts

```diff
@@ -11,12 +11,33 @@
 import 'core-js/stable';
 import 'regenerator-runtime/runtime';
 import path from 'path';
-import { app, BrowserWindow, shell, ipcMain } from 'electron';
+import {
+  app,
+  BrowserWindow,
+  shell,
+  ipcMain,
+  app as electronApp,
+} from 'electron';
 import { autoUpdater } from 'electron-updater';
 import log from 'electron-log';
 import MenuBuilder from './menu';
 import { resolveHtmlPath } from './util';
-
+import Realm from 'realm';
+
+const realmApp = new Realm.App('myapp-zufnj');
+
+async function logInRealm() {
+  const emailPasswordUserCredentials = Realm.Credentials.emailPassword(
+    'benperlmutter96@gmail.com',
+    'abc123'
+  );
+  try {
+    const user = await realmApp.logIn(emailPasswordUserCredentials);
+    return user;
+  } catch (err) {
+    console.error('log in error is...', err);
+  }
+}
```

Comment:
this function will need to be refactored as an IPC call to get data from renderer